README.md

```diff
@@ -849,3 +849,14 @@ In order to compile a production iOS build, run `npm run ios-build`, this will g
 
 #### Local production build the Android app
 To build an APK to share run (e.g. via Slack), run `npm run android-build`, this will generate a new APK in the `android/app` folder.
+
+# Onyx derived values
```

Comment:
Would you mind moving this to the section specifically about Onyx? https://github.com/Expensify/App?tab=readme-ov-file#onyx